stories/useMutationsHandler/usemutationshandler.guideline.mdx

```diff
@@ -0,0 +1,156 @@
+import * as Stories from './usemutationshandler.stories';
+import { Meta, Controls, Story, Canvas, Source } from '@storybook/blocks';
+
+<Meta name="Guideline" of={Stories} />
+
+# useMutationsHandler
+
+useMutationsHandler is a hook to simplify the handling of mutations and the display of feedback to the user.
+It provide a way to define a callback to call on success and a toast to display after the fail or success of a mutation.
+It also allows to define a single toast and callback for multiple related mutations instead of managing each mutation individually.
+
+## Properties
+
+<Controls of={Stories.Default} />
+
+## Required properties
+
+Only two properties are required to use this hook : mainMutation ans messageDescriptionBuilder.
+Like all other hook, it should be called at the top of your component.
+
+### mainMutation
+
+The mainMutation object contains two properties : name and mutation.
+
+mutation is the call of useMutation hook from react-query, see here from more information about this hook.
+The result should at least contain these properties from the hook :
+
+- data: data returned on success
+- error: error message on error
+- isError: boolean;
+- isIdle: boolean;
+- isLoading: boolean;
+- isSuccess: boolean;
+
+### messageDescriptionBuilder
+
+The messageDescriptionBuilder takes as parameters an array of mutations and returns a React Node.
+This array is composed of objects derived from mainMutation result and the eventual result of the mutations from dependentMutation array.
+
+This allows the rendering of a single message on success or failure, even when there are multiple mutations.
+As the return type is a React node it is possible to display simple message as well as more complex and detailed message.
+
+### Minimal Example
```

Comment:
I'll call this part `Example` or `Usage`. And I'll give a really complete example of use. The aim here is to provide the developer with a complete example of use. Like here, for example: https://github.com/scality/react-chained-query